### PR TITLE
Bulk Variable Updates to allow transactional rollbacks

### PR DESCRIPTION
**Pull request by richa-sawant**
Hello, have made modifications to the airflow/cli/commands/variable_command.py file. Kindly review 
Regards,
Richa

### CODE REVIEW

**Comment by richa-sawant** [comment]
Hi!
In this modification:

    I added a new option json_file to the variables_set function, which allows you to specify a JSON file.
    If json_file is provided, it loads the JSON file and iterates through the key-value pairs to set variables in the database.
    If any error occurs during the process, it rolls back the transaction and exits with an error message.

If any changes are to be done kindly let me know .

Richa :)

File: airflow/cli/commands/variable_command.py [hunk 1]
```diff
@@ -61,8 +61,25 @@ def variables_get(args):
 @providers_configuration_loaded
 def variables_set(args):
     """Create new variable with a given name, value and description."""
-    Variable.set(args.key, args.value, args.description, serialize_json=args.json)
-    print(f"Variable {args.key} created")
+    if args.json_file:
```
Comment (dirrao) [inline airflow/cli/commands/variable_command.py]: This functionality already exists. Check the CLI command as follows. 
I don't think this is needed. 

```
usage: airflow variables import [-h] file

Import variables

positional arguments:
  file        Import variables from JSON file

optional arguments:
  -h, --help  show this help message and exit
```

**Review by dirrao — CHANGES_REQUESTED** [review]
Thanks for your first MR. I have given my comment.

File: airflow/cli/commands/variable_command.py (hunk 1 above)
Comment (richa-sawant) [reply airflow/cli/commands/variable_command.py]: Hey ...thank you for the feedback! Have made the changes . Kindly review

Comment (potiuk) [reply airflow/cli/commands/variable_command.py]: What @dirrao suggests is that you do not need the change AT ALL. Why woul  you want to run CLI command inside another cli command? that makes no sense.

**Comment by potiuk** [comment]
Closing it - this change is just proxying existing command and makes little sense.

**Pull request closed without merging**